mesheryctl/pkg/utils/helpers.go

```diff
@@ -827,3 +806,15 @@ func SetOverrideValues(ctx *config.Context, mesheryImageVersion string) map[stri
 
 	return valueOverrides
 }
+
+// CheckFileExists checks if the given file exists in system or not
+func CheckFileExists(name string) (bool, error) {
+	_, err := os.Stat(name)
+	if err == nil {
+		return true, nil
+	}
+	if errors.Is(err, os.ErrNotExist) {
+		return false, fmt.Errorf("%s does not exist", name)
+	}
+	return false, errors.Wrap(err, fmt.Sprintf("%s does not exist", name))
```

Comment:
should this error statement be something generic, as the ErrNotExist case is already handled in line:817
For e.g. Failed to read/fetch the file?

Reply:
Good point, will make the changes tomorrow.